daos/record.go

```diff
@@ -154,26 +154,25 @@ func (dao *Dao) FindUserRelatedRecords(user *models.User) ([]*models.Record, err
 		return nil, err
 	}
 
-	result := []*models.Record{}
+	result := make([]*models.Record, 0, len(collections))
```

Comment:
This may not be necessary, since the cap most of the cases will be always wrong and will have to expand anyway because we are filling the result with the found records. Let's keep the literal version (in my opinion is easier to read too).